src/bson.c

```diff
@@ -1285,6 +1283,8 @@ void phongo_zval_to_bson(zval *data, php_phongo_bson_flags_t flags, bson_t *bson
 	 * properties, we'll need to filter them out later. */
 	bool         ht_data_from_properties = false;
 
+	ZVAL_UNDEF(&obj_data);
```

Comment:
Alternatively, I could have initialized the PHP 7 zval with `{0}` and had it work with `Z_ISUNDEF()`, but that makes assumptions about its internals and we don't do it anywhere else.